Source/Scene/ConditionsExpression.js

```diff
@@ -45,8 +48,13 @@ define([
     function ConditionsExpression(conditionsExpression) {
         this._conditionsExpression = clone(conditionsExpression, true);
         this._conditions = conditionsExpression.conditions;
-        this._expression = conditionsExpression.expression;
 
+        // Insert expression to expressions if it exists
+        // Then evaluate using expressions throughout class
+        // this._expression has to stay in prototype for specs to keep passing, but it can be removed in the future.
+        this._expression = conditionsExpression.expression;
+        this._expressions = conditionsExpression.expressions || {};
```

Comment:
We try to avoid this type of syntax, and also the allocation. Instead use `defaultValue(conditionsExpression.expressions, defaultValue.EMPTY_OBJECT)`